spec/system/jobseekers_can_manage_their_saved_jobs_spec.rb

```diff
@@ -10,12 +10,64 @@
   context "when logged in" do
     before do
       login_as(jobseeker, scope: :jobseeker)
-      visit jobseekers_saved_jobs_path
     end
 
-    it "shows their saved jobs" do
-      # TODO: Implement me properly when saved jobs are implemented
-      expect(page).to have_content(I18n.t("jobseekers.saved_jobs.index.page_title"))
+    context "and there are saved jobs" do
+      let(:school) { create(:school) }
+      let(:vacancy_1) { create(:vacancy) }
+      let(:vacancy_2) { create(:vacancy) }
+      let(:expired_vacancy) { create(:vacancy, :expired) }
+
+      before do
+        vacancy_1.organisation_vacancies.create(organisation: school)
+        vacancy_2.organisation_vacancies.create(organisation: school)
+        expired_vacancy.organisation_vacancies.create(organisation: school)
+
+        jobseeker.saved_jobs.create(vacancy: vacancy_1)
+        jobseeker.saved_jobs.create(vacancy: vacancy_2)
+        jobseeker.saved_jobs.create(vacancy: expired_vacancy)
+
+        visit jobseekers_saved_jobs_path
+      end
+
+      context "and they view saved jobs" do
```

Comment:
Last small thing, context should always start with `when` or `with`

Reply:
Yes I agree, thanks